### ISSUE TITLE
Add texture ignore tags to GameConfig export

### ISSUE THREAD

**Issue by yankscally** [QodotPlugin/Qodot#116]
someone mentioned that you can exclude filenames in your FGD with something like this:
```
    "textures": {
        "root": "textures",
        "extensions": [ ".png" ],
        "excludes": [ "*sky_bk", "*sky_dn", "*sky_ft", "*sky_lf", "*sky_rt", "*sky_up", "*_bump", "*_gloss", "*_glow", "_luma", "*_norm" ]
```

It would be nice to be able to export your own ignore tags from qodot, it is probably needed to use PBR textures. 

This is only so you don't see them in TrenchBroom’s texture browser, which fills up rather quickly.

**Comment by RhapsodyInGeek** [comment]
This is from the GameConfig, not the FGD file. Should be pretty easy to pull off. Might tackle it later this week.

**Comment by yankscally** [comment]
> This is from the GameConfig, not the FGD file. Should be pretty easy to pull off. Might tackle it later this week.

ah yeah - I see that now, I just changed the title. I'm trying to make this Material Maker work quite flawlessly with qodot, and by changing only a couple things in each I'm able to just export and everything automagically builds with these new fancy materials.

I had to change this default value to make it work:

![image](https://github.com/QodotPlugin/Qodot/assets/27631913/58199e40-45a8-4b1e-9801-84fa27ecc345)

and i had to remove "_albedo" from Material Makers export flags so it saves as "tex.png" _not_ "tex_albedo.png", and the material will be called "texture.tres" because this app supports exporting to godot. 

![image](https://github.com/QodotPlugin/Qodot/assets/27631913/5ac02cd9-a804-4566-a7f7-350fd85cc45d)


so pretty nice compatibility already even without any changes, the only thing I couldn't do without manually editing a file is this, but there is a short section about the `excludes` key in the TB manual.

adding this tag is a mix of all the tags from qodot and material maker.

```
    "textures": {
        "package": { "type": "directory", "root": "textures" },
        "format": { "extensions": ["jpg", "jpeg", "tga", "png"],"format": "image"},
        "excludes": ["*_orm", "*_normal", "*_heightmap", "*_emission", "*_sss", "*_metallic", "*_roughness", "*_ao"],
        "attribute": "_tb_textures"
```
